src/commands/init.js

```diff
@@ -130,6 +140,12 @@ InitCommand.description = 'Initialise a new Checkly Project'
 
 InitCommand.flags = {
   force,
+  skipSetup: flags.boolean({
```

Comment:
The `force` flag and `skipSetup` flag have the same purpose. I would use `force` since is more standard in UNIX commands

Reply:
Ahh didnt see there was a force. But it doesn't seem to skip the init inquirer.js prompts, does it?

Reply:
It was there but was not implemented yet. The behaviour you add is fine but we can just replace `skipSetup` with `force`

Reply:
Okay great, ill adjust it :+1: